app/models/task.py

```diff
@@ -400,7 +408,7 @@ def handle_import(self):
         try:
             self.extract_assets_and_complete()
         except zipfile.BadZipFile:
-            raise NodeServerError("Invalid zip file")
+            raise NodeServerError(gettext("Invalid zip file"))
```

Comment:
```suggestion
            raise NodeServerError(gettext("Invalid ZIP file"))
```
Unless specifically .zip. it should be "compressed file".